erigon-lib/diskutils/diskutils.go

```diff
@@ -0,0 +1,10 @@
+//go:build !darwin
+
+package diskutils
+
+import "fmt"
+
+func MountPointForDirPath(dirPath string) string {
+	fmt.Println("Implemented only for darwin")
```

Comment:
1. it must be log.Debug()
2. it must have log prefix, like "[diskutils] " 
3. most of users to run us on linux